cfn/cfn.json

```diff
@@ -0,0 +1,375 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Metadata": {
+        "AWS::CloudFormation::Designer": {
```

Comment:
I guess not, technically, but i like to be able to sanity-check the template in the designer UI